cmds/core/lockmsrs/lockmsrs.go

```diff
@@ -79,7 +79,10 @@ func main() {
 		log.Fatalf("Only Intel CPUs supported, expected GenuineIntel, got %v", cpuid.VendorIdentificatorString)
 	}
 
-	cpus := msr.AllCPUs
+	cpus, err := msr.AllCPUs()
```

Comment:
msrs, err := msr.All

so it does not stutter?

Reply:
I don't agree. It's not immediately obvious from msr.All or msr.Glob what you're getting back

Reply:
Yeah I agree with chris here, msr.All sounds like all msrs.